database/sql/import-sql.sh

```diff
@@ -41,13 +42,14 @@ cd /srv/database/backups/
 
 # Parse through each file in the directory and use the file name to
 # import the SQL file into the database of the same name
-sql_count=$(ls -1 ./*.sql 2>/dev/null | wc -l)
+sql_count=$(ls -1 ./*.sql* 2>/dev/null | wc -l)
 if [ "$sql_count" != 0 ]
 then
-	for file in $( ls ./*.sql )
+	for file in $( ls ./*.sql* )
 	do
 		# get rid of the extension
 		pre_dot=${file%%.sql}
+		pre_dot=${file%%.gz}
```

Comment:
should these two lines be the other way around?

Reply:
I did a workaround for this